Framework/Algorithms/test/MultiplyDivideTest.in.h

```diff
@@ -1050,6 +1050,7 @@ class @MULTIPLYDIVIDETEST_CLASS@ : public CxxTest::TestSuite
     auto rhs = create_RaggedWorkspace();
     auto result = performTest(lhs, rhs, false, DO_DIVIDE ? 1.0 : 4.0, DO_DIVIDE ? 1.4142135625 : 5.6568542436);
     TS_ASSERT(result->isRaggedWorkspace());
+    TS_ASSERT(result->isDistribution());
```

Comment:
Add an extra check for the units on divide to be empty and the same for everything else